src/ST/Client/WebApplicationRunner.cs

```diff
@@ -110,7 +110,10 @@ public void Start()
             var persistence = _container.GetInstance<IPersistenceController>();
             persistence.StartWatching(context.SpecPath);
             context.AddRemoteListener(persistence);
-
+
+            var dsl = _container.GetInstance<IFixtureController>();
```

Comment:
Not something to worry about now, but I've been tempted to try to remove the StructureMap dependency. Feels like it's more trouble than it's worth w/o fubu involved any longer.